craft_application/application.py

```diff
@@ -53,8 +53,10 @@
 class AppFeatures:
     """Specific features that can be enabled/disabled per-application."""
 
+    # Support for build-time secrets
     build_secrets: bool = False
-    """Support for build-time secrets."""
```

Comment:
Please keep the string, it's used to document the parameters
Like [this](https://github.com/canonical/craft-cli/blob/main/craft_cli/dispatcher.py#L40) becomes [this](https://canonical-craft-cli.readthedocs-hosted.com/en/latest/craft_cli/#craft_cli.CommandGroup)

Reply:
OK

Reply:
why was the trailing `.` dropped?